test/helper/helper.go

```diff
@@ -188,6 +188,33 @@ func WaitForDeviceState(t *testing.T, deviceId string, states ...metalv1.DeviceS
 	return false, fmt.Errorf("timed out waiting for device %v state %v to become one of %v", deviceId, device.GetState(), states)
 }
 
+func WaitForVrfRouteState(t *testing.T, routeId string, statuses ...metalv1.VrfRouteStatus) bool {
+	var route *metalv1.VrfRoute
+	var err error
+	t.Helper()
+	predefinedTime := 900 * time.Second // Adjust this as needed
```

Comment:
I observed this test passing at least once locally, but in CI it's timed out twice in a row.  Can it really take more than 15 minutes for a VRF route to finish updating or are we doing something wrong here?

Reply:
What was the pending status during the failure?

Reply:
Per offline conversation, the VRF route status will be pending until some other setup is done.  The pending status should not be used to determine if an update request can be made.  There is a hard-coded 5-minute wait before an update request is allowed, and the tests have been updated to match that.

It's also possible this entire VRF route subcommand should be removed.